src/mono/mono/mini/interp/interp-internals.h

```diff
@@ -97,7 +97,11 @@ typedef enum {
 	IMETHOD_CODE_UNKNOWN
 } InterpMethodCodeType;
 
+#ifdef HOST_WASM
+#define PROFILE_INTERP 1
+#else
```

Comment:
This needs to be controlled by an ifdef since it might have a significant runtime cost, i.e. the call mono_wasm_timestamp();

Reply:
yes, I want to make it into something which could be turned on on dev machine with wasm workload.

Reply:
Enabling it by default in debug builds should be ok, not sure whenever we have/use debug builds of the runtime.

Reply:
@lewing said "we could start producing debug runtime packs, open new issue"

But I think I would rather also have it in the release mode with empty stubs linked in. That should have no CPU cost, right ?

Reply:
And also, I want to do the same for AOT method frames, so far have no clue how.

Reply:
In theory, if the app is rebuilt, and the stubs are empty, then emscripten can optimize out the calls.